h/schemas/admin_group.py

```diff
@@ -61,6 +82,15 @@ class CreateAdminGroupSchema(CSRFSchema):
         title=_('Description'),
         description=_('Optional group description'),
         validator=colander.Length(max=GROUP_DESCRIPTION_MAX_LENGTH),
-        widget=TextInputWidget(rows=3),
+        widget=TextAreaWidget(rows=3),
+        missing=None
+    )
+
+    origins = colander.SchemaNode(
+        colander.String(),
+        title=_('Scope Origins'),
```

Comment:
I suspect from some initial chatter in Slack that the term "Scope Origins" might be confusing. Perhaps "URL prefixes" or "Websites" might be better? We can do some "hallway testing" with eg. Heather or Arti once this is up on prod.